File: netflix-genre.py
#!/usr/bin/env python3.7
# coding: utf-8
"""
    Generate Netflix Genre
    Handy tool to save your time to choose movie genre
"""


import random
import sys

def select_genre_word(filename):
    """
    Select Random Genre from File
    """
    with open(filename, 'r') as fname:
        return random.choice(fname.readlines()).replace('\n', '')

def generate_genre():
    """
    Generate Netflix Movie Genre
    """
    GENRE       = select_genre_word("genre")
    GENRE_WORD  = GENRE
    print("We will watch NetFlix Movie from Genre :", GENRE_WORD)


#############################################
### 
if __name__ == "__main__":
    generate_genre()
else:
    print("Not a Script. Exiting now...")
    sys.exit(2)

File: netflix-genre.go
package main

import (
	"bufio"
	"fmt"
	"math/rand"
	"os"
	"time"
)

func check(e error) {
	if e != nil {
		panic(e)
	}
}

func randomInt(min, max int) int {
	return min + rand.Intn(max-min)
}

func main() {
	var lastLine int
	rand.Seed(time.Now().UnixNano())
	randomNumber := randomInt(1, 18)

	fileName, err := os.Open("genre")
	check(err)

	scanner := bufio.NewScanner(fileName)
	for scanner.Scan() {
		lastLine++
		if lastLine == randomNumber {
			line := scanner.Text()
			fmt.Println("We will watch NetFlix Movie from Genre :", line)
		}
	}
}
